{
  "uid" : "3dfa8588b628afb8",
  "name" : "test_caller_approval_affected",
  "fullName" : "tests.brownie.token.tests.test_transferFrom#test_caller_approval_affected",
  "historyId" : "d5548e59afb70cb78e10802d517d851a",
  "time" : {
    "start" : 1642030634051,
    "stop" : 1642030634051,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "ValueError: connection already closed",
  "statusTrace" : "brownie_project = <Project 'TokenProject'>\naccounts = <brownie.network.account.Accounts object at 0x7f8ff5e1ac10>\n\n    @pytest.fixture(scope=\"module\")\n    def token(brownie_project, accounts):\n>       return brownie_project.Token.deploy(\"Test Token\", \"TST\", 18, 1e21, {'from': accounts[0]})\n\ntests/brownie/token/tests/conftest.py:49: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/contract.py:600: in __call__\n    return tx[\"from\"].deploy(\n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/account.py:508: in deploy\n    receipt, exc = self._make_transaction(\n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/account.py:725: in _make_transaction\n    raise VirtualMachineError(e) from None\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = VirtualMachineError(ValueError({'code': -32000, 'message': 'connection already closed'}))\nexc = {'code': -32000, 'message': 'connection already closed'}\n\n    def __init__(self, exc: ValueError) -> None:\n        try:\n            exc = exc.args[0]\n        except Exception:\n            pass\n    \n        if isinstance(exc, dict) and \"message\" in exc:\n            if \"data\" not in exc:\n>               raise ValueError(exc[\"message\"]) from None\nE               ValueError: connection already closed\n\n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/exceptions.py:85: ValueError",
  "flaky" : false,
  "newFailed" : false,
  "newBroken" : false,
  "newPassed" : false,
  "retriesCount" : 0,
  "retriesStatusChange" : false,
  "beforeStages" : [ {
    "name" : "configure_neon_network",
    "time" : {
      "start" : 1642030616111,
      "stop" : 1642030616111,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "brownie_project",
    "time" : {
      "start" : 1642030616843,
      "stop" : 1642030630059,
      "duration" : 13216
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "setup_env",
    "time" : {
      "start" : 1642030630972,
      "stop" : 1642030630972,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "accounts",
    "time" : {
      "start" : 1642030630972,
      "stop" : 1642030631271,
      "duration" : 299
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "token",
    "time" : {
      "start" : 1642030633473,
      "stop" : 1642030633823,
      "duration" : 350
    },
    "status" : "broken",
    "statusMessage" : "ValueError: connection already closed\n",
    "statusTrace" : "  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/_pytest/fixtures.py\", line 932, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/neon-compatibility/neon-compatibility/tests/brownie/token/tests/conftest.py\", line 49, in token\n    return brownie_project.Token.deploy(\"Test Token\", \"TST\", 18, 1e21, {'from': accounts[0]})\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/contract.py\", line 600, in __call__\n    return tx[\"from\"].deploy(\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/account.py\", line 508, in deploy\n    receipt, exc = self._make_transaction(\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/account.py\", line 725, in _make_transaction\n    raise VirtualMachineError(e) from None\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/exceptions.py\", line 85, in __init__\n    raise ValueError(exc[\"message\"]) from None\n",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  } ],
  "afterStages" : [ {
    "name" : "accounts::0",
    "time" : {
      "start" : 1642030678221,
      "stop" : 1642030678221,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "feature",
    "value" : "Brownie"
  }, {
    "name" : "parentSuite",
    "value" : "tests.brownie.token.tests"
  }, {
    "name" : "suite",
    "value" : "test_transferFrom"
  }, {
    "name" : "host",
    "value" : "fv-az28-122"
  }, {
    "name" : "thread",
    "value" : "3587-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.brownie.token.tests.test_transferFrom"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 23,
        "skipped" : 0,
        "passed" : 66,
        "unknown" : 0,
        "total" : 89
      },
      "items" : [ {
        "uid" : "62c7bc55050aee26",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/476//#testresult/62c7bc55050aee26",
        "status" : "broken",
        "statusDetails" : "ValueError: connection already closed",
        "time" : {
          "start" : 1642028350894,
          "stop" : 1642028350894,
          "duration" : 0
        }
      }, {
        "uid" : "c1d053687aa3d046",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/474//#testresult/c1d053687aa3d046",
        "status" : "broken",
        "statusDetails" : "ValueError: connection already closed",
        "time" : {
          "start" : 1642025673005,
          "stop" : 1642025673005,
          "duration" : 0
        }
      }, {
        "uid" : "af4763fb9eba90df",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/472//#testresult/af4763fb9eba90df",
        "status" : "broken",
        "statusDetails" : "ValueError: connection already closed",
        "time" : {
          "start" : 1642018238572,
          "stop" : 1642018238572,
          "duration" : 0
        }
      }, {
        "uid" : "61e120aca9b89fb8",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/471//#testresult/61e120aca9b89fb8",
        "status" : "broken",
        "statusDetails" : "ValueError: connection already closed",
        "time" : {
          "start" : 1642015604712,
          "stop" : 1642015604712,
          "duration" : 0
        }
      }, {
        "uid" : "1a517bd7e1f85070",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/470//#testresult/1a517bd7e1f85070",
        "status" : "broken",
        "statusDetails" : "ValueError: connection already closed",
        "time" : {
          "start" : 1642014230819,
          "stop" : 1642014230819,
          "duration" : 0
        }
      }, {
        "uid" : "e056d5630de65e0d",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/469//#testresult/e056d5630de65e0d",
        "status" : "broken",
        "statusDetails" : "ValueError: connection already closed",
        "time" : {
          "start" : 1642012912282,
          "stop" : 1642012912282,
          "duration" : 0
        }
      }, {
        "uid" : "d79eacf1d4426d5f",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/468//#testresult/d79eacf1d4426d5f",
        "status" : "broken",
        "statusDetails" : "ValueError: connection already closed",
        "time" : {
          "start" : 1642011422989,
          "stop" : 1642011422989,
          "duration" : 0
        }
      }, {
        "uid" : "e1fded3f09cdc9d1",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/466//#testresult/e1fded3f09cdc9d1",
        "status" : "passed",
        "time" : {
          "start" : 1641998000953,
          "stop" : 1641998005395,
          "duration" : 4442
        }
      }, {
        "uid" : "c2e7ceae14a98b23",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/465//#testresult/c2e7ceae14a98b23",
        "status" : "passed",
        "time" : {
          "start" : 1641993762145,
          "stop" : 1641993767130,
          "duration" : 4985
        }
      }, {
        "uid" : "3a9dda71ddcfcdba",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/464//#testresult/3a9dda71ddcfcdba",
        "status" : "passed",
        "time" : {
          "start" : 1641991019170,
          "stop" : 1641991023078,
          "duration" : 3908
        }
      }, {
        "uid" : "ac4f8b39b09c68ee",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/463//#testresult/ac4f8b39b09c68ee",
        "status" : "passed",
        "time" : {
          "start" : 1641986372275,
          "stop" : 1641986375089,
          "duration" : 2814
        }
      }, {
        "uid" : "a89dc3e27f8258a7",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/461//#testresult/a89dc3e27f8258a7",
        "status" : "broken",
        "statusDetails" : "ValueError: Gas estimation failed: 'HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1895fe5e80>: Failed to establish a new connection: [Errno 111] Connection refused'))'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.",
        "time" : {
          "start" : 1641982478604,
          "stop" : 1641982478604,
          "duration" : 0
        }
      }, {
        "uid" : "91452eb786912891",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/460//#testresult/91452eb786912891",
        "status" : "broken",
        "statusDetails" : "ValueError: Gas estimation failed: 'HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f189460db20>: Failed to establish a new connection: [Errno 111] Connection refused'))'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.",
        "time" : {
          "start" : 1641980267652,
          "stop" : 1641980267652,
          "duration" : 0
        }
      }, {
        "uid" : "16a671dcb0c511e9",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/459//#testresult/16a671dcb0c511e9",
        "status" : "passed",
        "time" : {
          "start" : 1641975836800,
          "stop" : 1641975840546,
          "duration" : 3746
        }
      }, {
        "uid" : "e54db66a8aff99ba",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/458//#testresult/e54db66a8aff99ba",
        "status" : "broken",
        "statusDetails" : "ValueError: Gas estimation failed: 'HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f189a842400>: Failed to establish a new connection: [Errno 111] Connection refused'))'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.",
        "time" : {
          "start" : 1641974827707,
          "stop" : 1641974827707,
          "duration" : 0
        }
      }, {
        "uid" : "ea1fc16f680c766c",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/457//#testresult/ea1fc16f680c766c",
        "status" : "broken",
        "statusDetails" : "ValueError: Gas estimation failed: 'HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1896042940>: Failed to establish a new connection: [Errno 111] Connection refused'))'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.",
        "time" : {
          "start" : 1641974206253,
          "stop" : 1641974206253,
          "duration" : 0
        }
      }, {
        "uid" : "635bb12ceee944a6",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/456//#testresult/635bb12ceee944a6",
        "status" : "broken",
        "statusDetails" : "requests.exceptions.ReadTimeout: HTTPConnectionPool(host='proxy.night.stand.neontest.xyz', port=80): Read timed out. (read timeout=30)",
        "time" : {
          "start" : 1641958010676,
          "stop" : 1641958010676,
          "duration" : 0
        }
      }, {
        "uid" : "acfd294b1fad5e32",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/455//#testresult/acfd294b1fad5e32",
        "status" : "broken",
        "statusDetails" : "requests.exceptions.ReadTimeout: HTTPConnectionPool(host='proxy.night.stand.neontest.xyz', port=80): Read timed out. (read timeout=30)",
        "time" : {
          "start" : 1641952206925,
          "stop" : 1641952206925,
          "duration" : 0
        }
      }, {
        "uid" : "6a63b975498c1828",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/454//#testresult/6a63b975498c1828",
        "status" : "broken",
        "statusDetails" : "requests.exceptions.ReadTimeout: HTTPConnectionPool(host='proxy.night.stand.neontest.xyz', port=80): Read timed out. (read timeout=30)",
        "time" : {
          "start" : 1641950684732,
          "stop" : 1641950684732,
          "duration" : 0
        }
      }, {
        "uid" : "13e5bfa34e7357e5",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/453//#testresult/13e5bfa34e7357e5",
        "status" : "broken",
        "statusDetails" : "ValueError: ['Program 53DfF883gyixYNXnM7s5xhdeyV8mVk9T4i2hGV9vG9io invoke [1]', 'Program log: program/src/storage_account.rs:76', 'Program log: program/src/account_data.rs:225', 'Program log: program/src/entrypoint.rs:1251 : Invalid Ethereum transaction nonce: acc 21, trx 20', 'Program 53DfF883gyixYNXnM7s5xhdeyV8mVk9T4i2hGV9vG9io consumed 51427 of 500000 compute units', 'Program 53DfF883gyixYNXnM7s5xhdeyV8mVk9T4i2hGV9vG9io failed: invalid program argument', 'Program 53DfF883gyixYNXnM7s5xhdeyV8mVk9T4i2hGV9vG9io invoke [1]', 'Program log: program/src/storage_account.rs:76', 'Program log: program/src/account_data.rs:225', 'Program log: program/src/entrypoint.rs:1251 : Invalid Ethereum transaction nonce: acc 21, trx 20', 'Program 53DfF883gyixYNXnM7s5xhdeyV8mVk9T4i2hGV9vG9io consumed 51427 of 500000 compute units', 'Program 53DfF883gyixYNXnM7s5xhdeyV8mVk9T4i2hGV9vG9io failed: invalid program argument', 'Program 53DfF883gyixYNXnM7s5xhdeyV8mVk9T4i2hGV9vG9io invoke [1]', 'Program log: program/src/storage_account.rs:76', 'Program log: program/src/account_data.rs:225', 'Program log: program/src/entrypoint.rs:1251 : Invalid Ethereum transaction nonce: acc 21, trx 20', 'Program 53DfF883gyixYNXnM7s5xhdeyV8mVk9T4i2hGV9vG9io consumed 51427 of 500000 compute units', 'Program 53DfF883gyixYNXnM7s5xhdeyV8mVk9T4i2hGV9vG9io failed: invalid program argument', 'Program 53DfF883gyixYNXnM7s5xhdeyV8mVk9T4i2hGV9vG9io invoke [1]', 'Program log: program/src/storage_account.rs:76', 'Program log: program/src/account_data.rs:225', 'Program log: program/src/entrypoint.rs:1251 : Invalid Ethereum transaction nonce: acc 21, trx 20', 'Program 53DfF883gyixYNXnM7s5xhdeyV8mVk9T4i2hGV9vG9io consumed 51427 of 500000 compute units', 'Program 53DfF883gyixYNXnM7s5xhdeyV8mVk9T4i2hGV9vG9io failed: invalid program argument']",
        "time" : {
          "start" : 1641947815335,
          "stop" : 1641947815335,
          "duration" : 0
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "3dfa8588b628afb8.json",
  "parameterValues" : [ ]
}